AEPServices/Sources/storage/FileSystemNamedCollection.swift

```diff
@@ -49,7 +49,7 @@ class FileSystemNamedCollection: NamedCollectionProcessing {
                 do {
                     try updatedStorageData.write(to: fileUrl, options: .atomic)
                 } catch {
-                    Log.warning(label: self.LOG_TAG, "Error when writing to file: \(error)")
+                    Log.warning(label: self.LOG_TAG, "Error \(error) when writing \(key) to collection \(collectionName)")
```

Comment:
i would vote to put our dynamic content in single quotes so it's clear to the reader exactly where the error, key, and collection name begin and end.  should we also announce `key` (e.g. - `...when writing key '\(key)'`) like we are for the other variables and in the subsequent log?